src/rgw/rgw_s3select_private.h

```diff
@@ -32,7 +32,14 @@
 #include <liboath/oath.h>
 
 
+#pragma GCC diagnostic push
```

Comment:
Maybe just drop this commit and let Gal address s3select, since it was only warnings rather than errors.

Reply:
dropped, rebased and repushed.